scripts/create_test_requirements.sh

```diff
@@ -0,0 +1,4 @@
+# Create test requirements under test/requirements.txt using requirements.in
```

Comment:
I don't quite understand the purpose of this script. 
1) Why do you need to create a `test/requirements.txt`?
2) Should it be part of the repo if it's a generated file?

Reply:
with pip-tools you write your necessary packages and pip-compile generates a reqs.txt which have all dependencies and versions, later on you install the packages with the reqs.txt

Reply:
Hmm ok but since it's a generated file, we should not commit it

Reply:
It is a generated file but it should not be ignore,

Whenever we change requirements.in we will update requirements.txt with this script.
Users will install packages with the requirements.txt.